### ISSUE TITLE
Markdown warnings with generated doc

### ISSUE THREAD

**Issue by dcpc007** [terraform-docs/terraform-docs#524]
### Describe the bug

"terraform-docs markdown ." generate a doc with several errors from linters (visual code studio)
Mainly https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md033 (and md041)

### How can we reproduce it?

generate a doc on even a simpliest project.
Affected sections : at least Providers and Inputs, but not Resource.

### Environment information

https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md033
Debian 10 (Linux dc-debian 5.10.0-7-amd64 #1 SMP Debian 5.10.40-1 (2021-05-28) x86_64 GNU/Linux)

**Comment by khos2ow** [comment]
This is almost identical to #500. The warnings are mostly due to anchors which you can be suppressed by `anchor: false` (or `--anchor=false` via CLI). If there is a multi-line default variable value there will also be some html tags in input sections too (i.e. `<pre>` and `<be>`) which can be suppressed by `html: false` (or `--html=false`).

**Comment by dcpc007** [comment]
Then if i ask users to do a local linter check before commit, they all have to implement exception rules ? :-(
why not use markdown official recommandation ? (didn't understand the need to "break" this)

**Comment by khos2ow** [comment]
> they all have to implement exception rules ?

Not necessarily, you can create [`.terraform-docs.yml`](https://terraform-docs.io/user-guide/configuration/) with all the configurations you want and check that in. So all your users need to do is simply running `terraform-docs .`

**Comment by giovannifontanel** [comment]
In my opinion, I should have the possibility to use `--anchor=true` flag (default value), generating an output like this:

```
## Requirements

| Name | Version |
|------|---------|
| [terraform](#requirement\_terraform) | >= 0.14.6 |
```

because I have already specified I want only markdown syntax via `terraform-docs markdown` command.

Furthermore, the wrong output with HTML anchor syntax is also generated via `terraform-docs markdown --html=false`.

As workaround, to solve the marklint [MD033](https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md033), I have to loss "anchor" feature.
